src/ui-shell/sidenav/sidenav-item.component.ts

```diff
@@ -31,11 +31,19 @@ import { Router } from "@angular/router";
 
 		<ng-template #sidenavItemRouterTpl>
 			<a
+				[attr.title]="title ? title : null"
 				[routerLink]="route"
+ 				[relativeTo]="routeExtras?.relativeTo"
+				[queryParams]="routeExtras?.queryParams"
+				[fragment]="routeExtras?.fragment"
+				[queryParamsHandling]="routeExtras?.queryParamsHandling"
+				[preserveFragment]="routeExtras?.preserveFragment"
+				[skipLocationChange]="routeExtras?.skipLocationChange"
+				[replaceUrl]="routeExtras?.replaceUrl"
+				[state]="routeExtras?.state"
+				[info]="routeExtras?.info"
```

Comment:
```suggestion
```

@klaascuvelier Considering we are using Angular 14, NavigationExtras does not have `info` option. We should remove it for now and add it if it is requested by users. This is a just in case measure to prevent breaking existing apps.

Reply:
and this is why i was suggesting the directive that would add all the properties if they exists there

Reply:
Ah that's why my editor gave a squiggly line, but I couldn't figure out why because the documentation was pointing to it.
Indeed also clearly confirms the concern @Licen-it voiced. 

For now I removed the info property. Will look into the directive suggestion after this. Want to fix this regression and unblock us first.

Reply:
Yea I understand, it would simplify porting between the versions. It's best to make another PR for the directive targeting the next release branch.